packages/shared/lib/core/ledger/stores/ledger-prepared-output.store.ts

```diff
@@ -0,0 +1,4 @@
+import { OutputTypes } from '@iota/types'
+import { writable } from 'svelte/store'
+
+export const ledgerPreparedOutput = writable<OutputTypes>()
```

Comment:
I imagine this is initialized as `undefined` rather than `null`, although thinking it best to explicitly pass `null` since we've been doing that elsewhere.